feature_engine/selection/drop_psi_features.py

```diff
@@ -135,10 +132,12 @@ class DropHighPSIFeatures(BaseSelector):
         test) will be switched. This is important because the PSI is not symmetric,
         i.e., PSI(a, b) != PSI(b, a)).
 
-    threshold: float, default = 0.25.
+    threshold: float, str, default = 0.25.
         The threshold to drop a feature. If the PSI for a feature is >= threshold, the
         feature will be dropped. The most common threshold values are 0.25 (large shift)
         and 0.10 (medium shift).
+        If threshold='auto', it will be calculated based on size of dataset and
```

Comment:
Small tweak please:

If 'auto', the threshold will be calculated based on the size of the base and target dataset and the number of bins.